packages/neos-ui-ckeditor5-bindings/src/EditorToolbar/LinkButton.js

```diff
@@ -4,7 +4,7 @@ import {connect} from 'react-redux';
 import {$get, $transform} from 'plow-js';
 import LinkInput from '@neos-project/neos-ui-editors/src/Library/LinkInput';
 
-import {IconButton} from '@neos-project/react-ui-components';
+import {IconButton} from '@neos-project/react-ui-components/src';
```

Comment:
ill see if i can fix this differently, maybe via webpack alias?

that would have the benefit, that people dont need to know how to avoid bundling the components twice.
Also we wont disturb our efforts in: https://github.com/neos/neos-ui/pull/3465

Reply:
in theory this should work:

in our [root webpack config](https://github.com/neos/neos-ui/blob/f042f183858b56743465b5806ee978a89b8c001f/webpack.config.js#L28) add this line:

```js
  ...
  resolve: {
    alias: {
      '@neos-project/react-ui-components$': path.resolve(__dirname, './packages/react-ui-components/src'),
      // idk if the next line is needed
      '@neos-project/react-ui-components/$': path.resolve(__dirname, './packages/react-ui-components/src')
    },
  },
```

it would be cool if someone could try this out and revert https://github.com/neos/neos-ui/commit/b13d470cbceda5636aad5128af143bc49924e703

maybe you? @crydotsnake :D xD

(i dont have a webpack build setup atm ^^ and i am to proud to change that. esbuild is my baby)

Reply:
Oh, yep, that looks about right. Didn't think of that :) I'll test this.

Reply:
Works like a charm :) Thanks for the pointer @mhsdesign!

> // idk if the next line is needed

It's not :)

Reply:
Thank you for the implementation ;)